src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -26,7 +33,11 @@ export class SelectMaxBid extends React.Component<SelectMaxBidProps> {
       <Container>
         <Title style={Margins.m1}>Your max bid</Title>
 
-        <MaxBidPicker bids={bids} />
+        <MaxBidPicker
+          bids={bids}
+          selectedBidIndex={this.state.selectedBidIndex}
+          onSelectNewBidIndex={index => this.setState({ selectedBidIndex: index })}
```

Comment:
Is there any reason we can't just use `onValueChange` or `selectedValue ` directly, rather than adding a new interface?

```jsx
<MaxBidPicker 
  bids={bids}
  onValueChange={(_, index) => this.setState({ selectedBidIndex: index })}
  selectedValue={this.state.selectedBidIndex}
/>
```

Reply:
Mmmm that would be because I didn't know about them. Thanks!